openstack/networking/v2/extensions/lbaas_v2/certificates/requests.go

```diff
@@ -12,20 +12,25 @@ type ListOptsBuilder interface {
 }
 
 type ListOpts struct {
+	Marker      string `q:"marker"`
```

Comment:
Why have you moved those fields to the top?

Reply:
I've just sort like in hc doc, but I will move it back